### PR TITLE
zsh-syntax-highlighting: fix another typo

### PR DESCRIPTION
**Pull request by alyssais**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

See https://github.com/Homebrew/homebrew-core/pull/9204.

cc @nijikon.

### CODE REVIEW

File: Formula/zsh-syntax-highlighting.rb [hunk 1]
```diff
@@ -27,7 +27,7 @@ def caveats
 
       source ~/.zshrc
 
-    Additionally, if you receive "highlighters directory not found" error message,
```
Comment (rakkesh) [inline Formula/zsh-syntax-highlighting.rb]: The sentence here is fine, as it is.

File: Formula/zsh-syntax-highlighting.rb [hunk 2]
```diff
@@ -27,7 +27,7 @@ def caveats
...
+    Additionally, if you a receive "highlighters directory not found" error message,
```
Comment (nijikon) [inline Formula/zsh-syntax-highlighting.rb]: This looks like a typo, `Additionally, if you a receive "highlighters directory not found" error message` => `Additionally, if you receive a "highlighters directory not found" error message`, but actually I think that we are good with what is there now.

**Review by nijikon — CHANGES_REQUESTED** [review]

File: Formula/zsh-syntax-highlighting.rb (hunk 2 above)
Comment (alyssais) [inline Formula/zsh-syntax-highlighting.rb]: Oh no! \*hides\*

**Comment by nijikon** [comment]
This is trivial, closing.

**Pull request closed without merging**